docs/build/smart-contracts/example-contracts/workspace.mdx

```diff
@@ -10,7 +10,7 @@ import { getPlatform } from "@site/src/helpers/getPlatform";
 
 The [workspace example] demonstrates how multiple smart contracts can be developed, tested, and built side-by-side in the same Rust workspace.
 
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)][oigp]
+[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?repo=stellar/soroban-examples&editor=web)
```

Comment:
I think we should keep the "link definition" convention used here. We can change the `oigp` to something more "relevent" like `oidc` (open-in-dev-container)? but having it defined here like this makes it possible to re-use the link later on. like on line 28 of this file.

Reply:
Updated!